integration/form-test.ts

```diff
@@ -534,6 +666,26 @@ test.describe("Forms", () => {
     });
 
     test.describe("in a splat route", () => {
+      test("no action resolves relative to the closest route", async ({
+        page,
+      }) => {
+        let app = new PlaywrightFixture(appFixture, page);
+        await app.goto("/projects/blarg");
+        let html = await app.getHtml();
+        let el = getElement(html, `#${SPLAT_ROUTE_NO_ACTION}`);
+        expect(el.attr("action")).toMatch("/projects");
```

Comment:
I think given Ryan's explanation of the splat behavior with "." - these 2 tests should assert the path `/projects/blargh` now